packages/components/panel/src/panel.scss

```diff
@@ -1,104 +1,136 @@
+
+  background: var(--sl-elevation-surface-raised-default-idle);
+  border: var(--sl-color-border-bold) solid var(--sl-size-borderWidth-subtle);
+  border-radius: var(--sl-size-borderRadius-md);
   display: flex;
   flex-direction: column;
 }
 
-:host([collapsible][collapsed]) {
-  sl-icon {
-    rotate: 0deg;
-  }
+:host([no-border]) {
+  border: 0;
+}
+
+:host([elevation='raised']) {
+  background: var(--sl-elevation-surface-raised-alternative-idle);
+}
 
+:host([elevation='sunken']) {
+  background: var(--sl-elevation-surface-raised-sunken-idle);
+}
+
+:host([collapsible][collapsed]) {
   [part='body'] {
     grid-template-rows: 0fr;
   }
 }
 
-[part='header'] {
-  align-items: start;
-  display: grid;
-  flex: 1;
-  grid-template-columns: minmax(min-content, max-content) minmax(0, auto);
-  justify-content: space-between;
+:host([collapsible]) {
+  [part='header'] {
+    grid-template-areas: 'toggle wrapper aside';
+    grid-template-columns: auto minmax(auto, 1fr) minmax(40px, auto);
+  }
+}
 
-  > * {
-    // FIXME: This is a quick hack to vertically align the heading with any action buttons
-    padding-block-start: 2px;
+:host([collapsible][toggle-placement='end']) {
+  [part='header'] {
+    grid-template-areas: 'wrapper aside toggle';
+    grid-template-columns: 1fr minmax(0, auto) auto;
   }
 }
 
-[part='wrapper'] {
-  align-items: center;
+:host([no-padding]) {
+  [part='content'] {
+    padding: 0;
+  }
+}
+
+:host([no-header]) {
+  [part='header'] {
+    padding: 0;
+  }
+}
+
+[part='titles'] {
   display: flex;
-  gap: 0.5rem;
-  padding-block: var(--_padding-block);
-  padding-inline: calc(var(--_padding-inline) + var(--_wrapper-padding-inline));
+  flex-direction: var(--sl-panel-titles-order);
+  gap: var(--sl-space-new-sm);
 }
 
-button {
-  appearance: none;
-  background: transparent;
-  border: 0;
-  border-radius: var(--sl-border-radius-button-md);
-  cursor: pointer;
+[part='header'] {
+  align-items: center;
+  display: grid;
   flex: 1;
-  outline: 0;
-  padding: 0;
-  text-align: start;
-
-  &:focus-visible {
-    outline: var(--_focus-outline);
-    outline-offset: var(--_focus-outline-offset);
-  }
+  gap: var(--sl-space-new-lg);
+  grid-template-areas: 'wrapper aside';
+  grid-template-columns: 1fr minmax(0, auto);
+  padding: var(--sl-space-new-lg);
 }
 
-sl-icon {
-  rotate: 180deg;
-  transition: rotate 0.2s ease-in-out;
+[part='wrapper'] {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--sl-space-new-lg);
+  grid-area: wrapper;
 }
 
 slot[name='heading'],
 slot[name='heading']::slotted(*) {
-  color: var(--sl-color-accordion-default-foreground);
-  font: var(--sl-text-accordion-title);
+  color: var(--sl-color-text-bold);
+  font-size: 1.5em;
+  line-height: 1.5em;
 }
 
 slot[name='heading'] {
   display: block;
 }
 
-sl-tool-bar {
-  min-inline-size: 0;
+slot[name='subheading'],
+slot[name='subheading']::slotted(*) {
+  color: var(--sl-color-text-bold);
+  line-height: 1.5;
```

Comment:
you have no `em` here. But on lines 83, 84 you do. Should it not be the same everywhere?